requirements.txt

```diff
@@ -1,5 +1,6 @@
+molo.core==8.0.2
```

Comment:
This is a duplicate requirement, molo.core is already required lower down.
Also this downgrades molo.core.